src/include/sof/trace/trace.h

```diff
@@ -198,14 +198,6 @@ void trace_init(struct sof *sof);
 	_trace_event_atomic_with_ids(class, id_0, id_1, 1, format,	\
 				     ##__VA_ARGS__)
 
-/* component tracing builder macro */
```

Comment:
The commit message explains why we make the change, but it does not explain what the change is i.e. are we moving trace defs around or are we modifiying the definitions ?

Reply:
added